docs/cvl/cvl2/changes.md

```diff
@@ -0,0 +1,583 @@
+Syntax changes introduced in CVL 2
+==================================
+
+CVL 2.0 is a major overhaul to the type system of CVL.  Many
```

Comment:
I'd argue it's gone beyond the type system. I would say it's an overhaul to the "internals" and leave it at that.

Reply:
oh you literally say that on the next line. well, never mind then, leave as is